pkg/portal/smtp/service.go

```diff
@@ -74,7 +75,9 @@ func (s *Service) SendTestEmail(ctx context.Context, app *model.App, options Sen
 
 	err = dialer.DialAndSend(message)
 	if err != nil {
-		return
+		return errors.Join(SMTPTestFailed.NewWithInfo("smtp test failed", map[string]interface{}{
+			"Message": err.Error(),
+		}), err)
```

Comment:
What about just use `err.Error()` as message?

Reply:
Currently the joined error message will be returned as `message` so I didn't use it.
Added a commit to remove internal errors from the graphql error, so that message can be directly used.